src/components/templates/ArtPiece/ArtPiece.tsx

```diff
@@ -31,11 +31,11 @@ const ArtPiece = () => {
       <div className="full-page-container art-piece-container">
         <InformationLeftColumn venueLogoPath={venue ? venue.host.icon : ""}>
           <InformationCard title="About the venue">
-            <p>
-              {venue.name}.
-              <br />
-              {` It's a real piece of art.`}
+            <p style={{ fontSize: 22, fontWeight: "bold" }}>{venue.name}.</p>
```

Comment:
Can you create classes for this

Reply:
changed it on the other PR

Reply:
The parent class fixes the font to 12px, but I don't wanna change parent classes because they are also used in the JazzBar. Can I not leave it like this?

Reply:
I think Mathieu means create a class in the .scss file (or create a style of type Record<string, CSSProperties> outside the component) and place the style object in there.